### PR TITLE
Add error codes

### PR DESCRIPTION
**Pull request by superpowers04**
Add some error codes, enhance some of the page UI and add some pages

### CODE REVIEW

**Comment by superpowers04** [comment]
Pull request now should be good to go

File: _help/exit-code/code-1073740777/index.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Windows 7 / 8
+
+- In Windows 7 click on the Start button to open your Start Menu. When the Start Menu opens click on the Control Panel menu option. In Windows 8, click on the Control Panel app in the Windows 8 Start Screen. Then scroll to the bottom and click on the More Settings option. 
+- When the Control Panel window opens click on the Uninstall a program option under the Programs category. This option is shown by the arrow in the image below. If you are using the Classic View of the Control Panel, then you would double-click on the Programs and Features icon instead. 
+- Select the program you want to uninstall and then click the "uninstall" button.
+- Follow the instruction to uninstall D3DGear step by step. 
+- Restart your PC if required.
+
+## Windows Vista
```
Comment (LordRalex) [inline _help/exit-code/code-1073740777/index.markdown:21]: Remove Vista and XP. They are not supported.

File: _help/exit-code/code-1073740777/index.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+name: "ec-1073740777"
+desc: "Exit Code: -1073740777"
+---
+
+# Exit Code: -1073740777
+If you are receiving this error, the issue is most-likley due to having software known as "D3Dgear" running on your computer.
+How to fix this
+Currently, the only way to fix this is by completely uninstalling the program.
+
+## Windows 7 / 8
```
Comment (LordRalex) [inline _help/exit-code/code-1073740777/index.markdown:13]: We also need Windows 10.

File: _help/exit-code/code-1073741819/index.markdown [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
+# Exit Code: -107374181 <small>(HEX: 0xC0000005 - STATUS_ACCESS_VIOLATION)</small>
+We do not yet know the root cause of this issue. In order to solve this issue, some known workarounds include reinstalling the JVM or updating your graphics drivers, this error may also be caused by having "D3Dgear" installed.
+
+## How to fix this
+Due to the quite vagueness of the error, we are unsure of the exact cause. Please try the following and attempt to see it they solve your case:
+
+* This may possibly be caused by having D3Dgear installed. (If that is the case, please let us know!) 
+* Try updating your graphics drivers.
+
```
Comment (LordRalex) [inline _help/exit-code/code-1073741819/index.markdown:15]: Remove the extra lines here.

File: _help/exit-code/code-1073741819/index.markdown [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+name: "ec-107374181"
+desc: "Exit Code: -107374181"
+---
+# Exit Code: -107374181 <small>(HEX: 0xC0000005 - STATUS_ACCESS_VIOLATION)</small>
+We do not yet know the root cause of this issue. In order to solve this issue, some known workarounds include reinstalling the JVM or updating your graphics drivers, this error may also be caused by having "D3Dgear" installed.
+
+## How to fix this
+Due to the quite vagueness of the error, we are unsure of the exact cause. Please try the following and attempt to see it they solve your case:
+
+* This may possibly be caused by having D3Dgear installed. (If that is the case, please let us know!)
```
Comment (LordRalex) [inline _help/exit-code/code-1073741819/index.markdown:13]: Link the D3DGear page.

File: _help/exit-code/code-2147483645/index.markdown [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+title:  "Exit Code: -2147483645"
+name: "ec-2147483645"
+desc: "Exit Code: -2147483645"
+---
+# Exit Code: -2147483645 <small>(HEX: 0x80000003 - One or more arguments are invalid)</small>
+This error is caused when the registry contains malformed information involving the current Java installation. In most cases, reinstalling the proper version of Java should solve this, otherwise, you should get a registry cleaner to scan your computer.
+
+## Reinstalling Java
+Minecraft, runs on a platform called Java. Java comes in many shapes, and forms, which may make it confusing exactly which version of it you should have installed, which is what this should hopefully explain.
+[You can find info on how to reinstall java here](http://localhost:4000/help/how-to-reinstall-java/)
```
Comment (LordRalex) [inline _help/exit-code/code-2147483645/index.markdown:12]: This is a bad link.

File: _help/exit-code/index.markdown [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+            <th>Exit Code</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>
+                <a href="https://bugs.mojang.com/browse/MCL-90">MCL-90</a>
+            </td>
+            <td>
+                <a href="/MinecraftHopper/help/exit-code/code-139/">139</a>
```
Comment (LordRalex) [inline _help/exit-code/index.markdown:28]: All the URLS here are wrong, they should not have MinecraftHopper in them.

File: _help/failed-check-session-lock/index.markdown [hunk 7]
```diff
@@ -1,17 +1,24 @@
 ---
 layout: article
-title:  "Failed to Check Session Lock"
-name: "failed-check-session-lock"
-desc: "How to fix Failed to Check Session Lock in Minecraft"
+title:  "Exit code 139"
```
Comment (LordRalex) [inline _help/failed-check-session-lock/index.markdown:3]: You changed this page by accident.

File: _layouts/default.html [hunk 8]
```diff
@@ -19,10 +20,11 @@
             <li><a href="https://bugs.mojang.com/secure/Dashboard.jspa">Bug Tracker</a></li>
             <li><a href="http://help.minecraft.net/">Account Support</a></li>
         </ul>
-        <h3 class="muted"><a href="/">{{ site.title | default: site.github.repository_name }}</a></h3>{{ site.description | default: site.github.project_tagline }}
+        <a href="/" class="muted"><h3>MinecraftHopper</h3></a>Get community help for Minecraft
```
Comment (LordRalex) [inline _layouts/default.html:23]: Revert these changes, this file is okay as-is.

**Review by LordRalex — CHANGES_REQUESTED** [review]

File: _help/exit-code/code-1073741819/index.markdown (hunk 4 above)
Comment (LordRalex) [reply _help/exit-code/code-1073741819/index.markdown:11]: I had meant the removal page, not the main site. (exit code 1073740777)

File: _help/exit-code/index.markdown [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+title:  "Exit statuses"
+name: "exit-codes"
+desc: "Exit statuses"
+---
+
+
+# What is a Exit Code? <small>- Exit statuses</small>
+Exit statuses (or "Exit Codes") are integers that are returned when an application exits. These exit statuses can be caused by either Minecraft, or the operating system in use. There is no differentiation that can be made as to which is responsible for the Exit code, but currently Minecraft does not use non-generic exit statuses.
+
+Exit statuses are also OS-dependent integers unless set by Minecraft or a another loaded library.
```
Comment (LordRalex) [inline _help/exit-code/index.markdown:12]: "a another"

Comment (superpowers04) [reply _help/exit-code/index.markdown:12]: I never put that :thinking:

**Pull request merged**